package/MDAnalysis/transformations/rotate.py

```diff
@@ -76,31 +77,44 @@ def rotateby(angle, direction, center="geometry", pbc=None, ag=None, position=No
         or 'mass'
     pbc: bool or None, optional
         If True, all the atoms from the given AtomGroup will be moved to the unit cell
-        before calculating the center
-    position: list, optional
+        before calculating the center. Warning: Wrapping/unwrapping the trajectory or 
+        performing PBC corrections may not be possible after rotating the trajectory. 
+    position: array-like, optional
```

Comment:
To be consistent with the other transformation, the option should be named `point`. The name of the user facing variable is more important than the name of the variable we manipulate inside; so the variable in the wrapper should be renamed so the argument is consistent to the user.